liwords-ui/src/shared/usernameWithContext.tsx

```diff
@@ -180,7 +180,7 @@ export const UsernameWithContext = (props: UsernameWithContextProps) => {
       overlay={userMenu}
       getPopupContainer={() => document.getElementById('root') as HTMLElement}
       placement="bottomLeft"
-      trigger={['click']}
+      trigger={!loggedIn && props.omitProfileLink ? [] : ['click']}
```

Comment:
breaks https://woogles.io/profile/HastyBot (#891) when not logged in.

also, mouseover cursor is still suggesting it's clickable.